tests_new/integration_tests/client.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import boto3
+import os
+from munch import DefaultMunch
+
+
+ENVNAME = os.getenv('ENVNAME', 'dev')
+
+
+class Client:
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+        self.token = self._get_jwt_token()
+
+    def query(self, query: str):
+        graphql_endpoint = f'{os.getenv("API_ENDPOINT", False)}graphql/api'
+        headers = {'AccessKeyId': 'none', 'SecretKey': 'none', 'authorization': self.token}
+        r = requests.post(graphql_endpoint, json=query, headers=headers)
+        r.raise_for_status()
+
+        return DefaultMunch.fromDict(r.json())
+
+    def _get_jwt_token(self):
+        cognito_client = boto3.client('cognito-idp', region_name=os.getenv('AWS_REGION', 'eu-west-1'))
```

Comment:
Can't we authenticate same way as we do with the browser? 
Why do we need the boto3 calls? 
Also this ties the authentication to Cognito, what about customers that they use a custom IdP?
Maybe something like [this](https://gist.github.com/miguelmota/8b519212aca47210d529532b3d8e5b2f)?

I guess this one will be solved by the SDK as well.

Reply:
Exactly I did not want to spend much time on something that will be replaced by the SDK/CLI authentication, because it is going to mimic it